<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Krypta Map Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="15,100,20">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QFrame" name="frameTool">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Kryed::GLPanel" name="glWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="utilLayout" stretch="33,33,33">
        <item>
         <widget class="QGroupBox" name="entityGroup">
          <property name="title">
           <string>Entity</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="entityLayout" stretch="80,10,10">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="lEntity">
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lEntityName">
               <property name="text">
                <string>None</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bBrowseEntities">
               <property name="text">
                <string>Browse entities</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="envGroup">
          <property name="title">
           <string>Environment</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="envLayout" stretch="80,10,10">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="lEnv">
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lEnvName">
               <property name="text">
                <string>None</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bBrowseEnv">
               <property name="text">
                <string>Browse env</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="layerGroup">
          <property name="title">
           <string>Layer</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="layerLayout">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QTableWidget" name="layerProperties">
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <property name="rowCount">
                <number>3</number>
               </property>
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>Descr</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Width</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Height</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Property</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>Description:</string>
                </property>
                <property name="textAlignment">
                 <set>AlignRight|AlignVCenter</set>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>Width:</string>
                </property>
                <property name="textAlignment">
                 <set>AlignRight|AlignVCenter</set>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>Height:</string>
                </property>
                <property name="textAlignment">
                 <set>AlignRight|AlignVCenter</set>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bLayerMan">
               <property name="text">
                <string>Layer manager</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="miFileNew"/>
    <addaction name="miFileOpen"/>
    <addaction name="miFileExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolMain">
   <property name="windowTitle">
    <string>Krypta Map Editor</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="miFileNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="miFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="miFileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Kryed::GLPanel</class>
   <extends>QWidget</extends>
   <header>glpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
